### PR TITLE
Fix ETFlag conditional

### PR DESCRIPTION
**Pull request by kdeloach**
I forgot to update this conditional when I added enums to the parser.

### CODE REVIEW

File: gwlfe/gwlfe.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def run(z):
 
                     # CALCULATE ET FROM SATURATED VAPOR PRESSURE,
                     # HAMON (1961) METHOD
-                    if z.ETFlag == 0:
+                    if z.ETFlag is ETflag.HAMMON_METHOD:
                         if z.DailyTemp > 0:
```
Comment (caseypt) [inline gwlfe/gwlfe.py]: Typo. Should be `HAMON`.

Comment (kdeloach) [inline gwlfe/gwlfe.py]: Haha that typo exists in the original VB code. Will fix it now.

File: gwlfe/enums.py [hunk 2]
```diff
@@ -18,14 +18,14 @@ def parse(value):
 
 
 class ETflag(object):
-    HAMMON_METHOD = '<Hammon method>'
+    HAMON_METHOD = '<Hammon method>'
```
Comment (caseypt) [inline gwlfe/enums.py]: So we should expect GMS files to use `Hammon`?

Comment (kdeloach) [inline gwlfe/enums.py]: The GMS file actually uses `0` to indicate the Hamon method. These string values exist purely for symbolic reasons. But you're right, this should be "Hamon".

**Comment by kdeloach** [comment]
Updated.

**Comment by caseypt** [comment]
+1.

**Pull request merged**